EnglishTranslation-master/QUEST_JOBSTEP.tsv

```diff
@@ -3084,55 +3084,55 @@ QUEST_JOBSTEP_20150730_003083	The Squire Master says he needs the Seal of the Le
 QUEST_JOBSTEP_20150803_003097	It's been recorded completely.{nl}Good. I will pass this to Vaidotas.{nl}Now is my turn to help you.
-QUEST_JOBSTEP_20150803_003098	The thoughts of the dead... All Necromancers want to possess them.{nl}Especially the keepsakes that lost their owners.{nl}
-QUEST_JOBSTEP_20150803_003099	A strong grudge is swirling around Videntis Shrine according to what I found out.{nl}If you could obtain those, I will open the door of a higher level Necromancer.
-QUEST_JOBSTEP_20150803_003100	If you will be able to pick out this kind of object, you will become a stronger Necromancer.{nl}For now, I will help you as a reward for the object.
-QUEST_JOBSTEP_20150803_003101	Ah, you came at the right time. {nl}I am investigating Crystal Mine, but it will be dangerous if we leave it like that so I want to take care of it quickly.{nl}
-QUEST_JOBSTEP_20150803_003102	If you could help my task, I think the investigation would be done sooner.{nl}And it will be also a great chance to test your skills as an Alchemist.
-QUEST_JOBSTEP_20150803_003103	I am glad that Vaidotas asked me a favor.{nl}Return with this.
-QUEST_JOBSTEP_20150803_003104	Good.{nl}I need the orb that records the memories of the past from the Chronomancer Master.{nl}He is the head of our Winterspoon Family so I've requested it to him in advance. Tell him that I've sent you.
-QUEST_JOBSTEP_20150803_003105	Lucid... Actually have you met the Chronomancer Master well?
-QUEST_JOBSTEP_20150803_003106	The orb reads the past memories of its owner and records them.{nl}Explore all areas in District 1 and 2 in the Crystal Mine with this.{nl}
-QUEST_JOBSTEP_20150803_003107	When you use this orb after exploring all regions, it will read the memories of the possessor and record the past.{nl}If you had explored the regions in the past, then record it into the orb right away.{nl}
-QUEST_JOBSTEP_20150803_003108	Well done. {nl}It will take some time to check this so in the meantime, I will teach you new alchemy.
+QUEST_JOBSTEP_20150803_003098	The thoughts of the dead...All Necromancers want to possess them.{nl}Especially those regarding lost keepsakes.{nl}
+QUEST_JOBSTEP_20150803_003099	According to what I found out, there's a strong grudge swirling around Videntis Shrine.{nl}If you could obtain whatever it is causing this grudge, I will open the door to a higher level of Necromancy.
+QUEST_JOBSTEP_20150803_003100	If you will be able to pick out this kind of object, you will become a stronger Necromancer.{nl}For now, I'll reward you if you can find one.
+QUEST_JOBSTEP_20150803_003101	Ah, you came at the right time. {nl}I am investigating Crystal Mine, but it will be dangerous if we leave it as is, so I want to take care of it quickly.{nl}
+QUEST_JOBSTEP_20150803_003102	If you could help me, I think the investigation would be done sooner.{nl}And it will be also a great chance to test your skills as an Alchemist.
```

Comment:
should remove `And` from the start of the second sentence, it's a bit unnecessary